ktor-server/ktor-server-tests/jvmAndNix/test/io/ktor/tests/server/plugins/CORSTest.kt

```diff
@@ -1007,6 +1007,30 @@ class CORSTest {
         }
     }
 
+    @Test
+    fun testCorsValidationWithTrailingSlashOrigin () = testApplication {
+        application {
+            install(CORS){}
+            routing {
+                get {
+                    call.respond(HttpStatusCode.OK)
+                }
+            }
+        }
+
+        client.get("/") {
+             header(HttpHeaders.Origin, "http://localhost:3000/")
+        }.let {
+            assertEquals(HttpStatusCode.Forbidden, it.status)
+        }
+
+        client.get("/") {
+            header(HttpHeaders.Origin, "http://localhost:3000")
+        }.let {
+            assertEquals(HttpStatusCode.Forbidden, it.status)
+        }
```

Comment:
Shouldn't this be...

    client.get("/") {
        header(HttpHeaders.Origin, "http://localhost:3000/")
    }.let {
        assertEquals(HttpStatusCode.OK, it.status)
    }

    client.get("/") {
        header(HttpHeaders.Origin, "http://localhost:3000")
    }.let {
        assertEquals(HttpStatusCode.OK, it.status)
    }

...?

Reply:
in this test CORS does not contain any settings(`install(CORS){}`) and the client host is different from the server host([allowSameOrigin](https://github.com/ktorio/ktor/blob/main/ktor-server/ktor-server-plugins/ktor-server-cors/jvmAndNix/src/io/ktor/server/plugins/cors/CORS.kt#L170) does not apply here)
[this check](https://github.com/ktorio/ktor/blob/main/ktor-server/ktor-server-plugins/ktor-server-cors/jvmAndNix/src/io/ktor/server/plugins/cors/CORS.kt#L171) will not pass because this host is not contained in CORS config hosts and it should return [403 forbidden](https://github.com/ktorio/ktor/blob/main/ktor-server/ktor-server-plugins/ktor-server-cors/jvmAndNix/src/io/ktor/server/plugins/cors/CORSUtils.kt#L77)
Am I missing something?

Reply:
Apologies, I misread the issue and assumed the test provided in the comments was the intended behaviour.